### DISCUSSION TITLE
How can I hide specific control icons?

### DISCUSSION THREAD

**Discussion by Alatarius** [FreeTubeApp/FreeTube#6025] [Q&A]
To be more specific I want to hide these icons (circled in red)

![Untitled](https://github.com/user-attachments/assets/1cfbe622-1fea-4399-9ee2-f10d5bc1004f)

I have no use for them personally and they become very distracting.

These have just appeared as of this newest update (0.22.0 Beta)

**Comment by absidue** [comment]
They all existed in the old player, they just have different icons than you remember as it is a different video player. From left to right they are playback rate menu, loop the current video and audio track selector.

And no you can't hide those icons.
